src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar calendar--start-day-sun calendar--month-length-31">
+      <article class="calendar__day"></article>
```

Comment:
For individual days in a calendar, it's better to use `<div>` instead of `<article>`, as `<article>` is meant for self-contained, independent content.
Also `calendar` - is not a `section`.


For you to understand better, for example


<!section - posts >
    < article >Post<!article >
    < article >Post<!article >
    < article >Post<!article >
<!section - posts>